src/components/PersonInfo/PersonInfo.tsx

```diff
@@ -0,0 +1,56 @@
+import { Link, useParams, useSearchParams } from 'react-router-dom';
+import { Person } from '../../types';
+import classNames from 'classnames';
+
+type Props = {
+  person: Person;
+  father?: Person;
+  mother?: Person;
+};
+
+export const PersonInfo: React.FC<Props> = ({ person, father, mother }) => {
+  const { slug } = useParams();
+  const [searchParams] = useSearchParams();
+
+  return (
+    <tr
+      key={person.slug}
```

Comment:
The `key` prop should not be used on the `<tr>` element directly within the component. The `key` should be applied to the component instance in the parent component where the list is being mapped over. Consider removing the `key` prop from here.